documentation/staging/content/userguide/managing-domains/model-in-image/auxiliary-images.md

```diff
@@ -123,6 +148,126 @@ respectively, and must be changed to specify a directory in
+
+The expected merge order for auxiliary images that share
+the same auxiliary image volume is:
+
+- If you specify auxiliary images at different `serverPod` scopes,
+  and they all share the same volume, then the files from the
+  domain scope will be merged first, the cluster scope second,
+  and the server scope last.
+
+- If you specify multiple auxiliary images at the same scope, 
+  and they all share the same volume, then the files
+  will be merged in the order in which images appear
+  under `serverPod.auxiliaryImages`.
+
+__Note:__ If the results of a merge yield two or more
+differently named `domain.spec.configuration.model.ModelHome` model files in
+an auxilliary image volume, then refer to
+[Model file naming and loading order]({{< relref "/userguide/managing-domains/model-in-image/model-files#model-file-naming-and-loading-order">}})
+for the model files loading order.
+
+#### Performing replaces instead of merges
+
+If multiple auxiliary images share the same volume
+and you prefer that a particular same named directory from a later
+image completely replaces a directory from a previous
+image instead of combining the two directories,
+then you can customize the command that the second image uses to populate
+the shared volume in order to force a replace.
+
+For example, you can customize the 
+the second image's `serverPod.auxiliaryImage.command` field
+to first delete the directory that was already copied from the
+earlier image and then have it perform a normal copy:
+
+```shell
+...
+spec:
+  ...
+  auxiliaryImageVolumes:
+  - name: aivolume
+    mountPath: /auxiliary
+  ...
+  serverPod:
+    auxiliaryImages:
+    - image: domain-image-A:v1
+      volume: aivolume
+    - image: domain-image-B:v1
+      volume: aivolume
+      # the following command replaces 'mydir' instead of merging it:
+      command: 'rm -fr $TARGET_MOUNT_PATH/mydir; cp -R $COMMON_MOUNT_PATH/* $TARGET_MOUNT_PATH'
+```
+
+#### Merge ordering example
+
+Assuming you have auxiliary images are defined at the domain, the WebLogic cluster,
+and the server scope (myserver is part of the mycluster):
+
+```
+...
+spec:
+  auxiliaryImageVolumes:
+  - name: aivolume
+    mountPath: /auxiliary
+  serverPod:
+    auxiliaryImages:
+    - image: domain-image-A:v1
+      volume: aivolume
+    - image: domain-image-B:v1
+      volume: aivolume
+  cluster:
+  - name: mycluster
+    serverPod:
+      auxiliaryImages:
+      - image: cl-image-A:v1
+        volume: aivolume
+      - image: cl-image-B:v1
+        volume: aivolume
+  managedServers:
+  - serverName: "myserver"
+    serverPod:
+      auxiliaryImages:
+      - image: ms-image-A:v1
+        volume: aivolume
+
+```
+
+Then the files from the images will be merged into their shared `aivolume` volume `/auxiliary` mount path in the following order:
+
+```
+domain-image-A:v1 (first)
+domain-image-B:v1
+cl-image-A:v1
+cl-image-B:v1
+ms-image-A:v1 (last)
+```
+
 ### Sample
```

Comment:
Under Step 2
Follow these steps to create a auxiliary image -> Follow these steps to create an auxiliary image
On line 625
custom file name in the above command. -> custom file name in the previous command.